include/bitcoin/bitcoin/wallet/select_outputs.hpp

```diff
@@ -29,16 +29,29 @@ namespace wallet {
 
 struct BC_API select_outputs
 {
-    enum class algorithm
+    /// Algorithm summary:
+    ///
+    /// greedy: returns the first available unspent output that is the
+    /// minimum greater than the specified amount if any.  if there
+    /// are none, it returns a set of the largest outputs (in desc
+    /// order to minimize the number of inputs) that are smaller than
+    /// the specified amount and the amount of change.
+    ///
+    /// individual: returns a set of individual unspent outputs that
+    /// satisfy the specified amount.  For example, setting amount to
+    /// 0 will return all unspent outputs since any of them can
+    /// satisfy that amount.  The change amount will always be 0.
+    enum class algorithm : uint8_t
     {
-        greedy
+        greedy,
+        individual
     };
 
     /// Select optimal outpoints for a spend from unspent outputs list.
     /// Return includes the amount of change remaining from the spend.
     static void select(chain::points_info& out,
-        chain::output_info::list unspent, uint64_t minimum_value,
-        algorithm option=algorithm::greedy);
+        chain::output_info::list unspent, const uint64_t minimum_value,
```

Comment:
`chain::output_info::list unspent` should be `const &` (same below).

Avoid `const` in by-value parameters (same below).